daffodil-cli/src/it/scala/org/apache/daffodil/generating/TestCLIGenerateC.scala

```diff
@@ -78,6 +83,8 @@ class TestCLIGenerateC {
     try {
       shell.sendLine(generateCmd)
       shell.expect(contains("Unknown option 's'"))
+
+      Util.expectExitCode(ExitCode.GenerateCodeError, shell)
```

Comment:
I think this and many other tests in this file should be an ExitCode.Usage error.

And looking at the CI, many of these tests are failing because they are looking for the wrong exit code.

Make sure you are running the tests to make sure they work before pushing them for review. To run these CLi integration tests, you need to run them with 

```
sbt IntegrationTest/test
```